### DISCUSSION TITLE
Why does `misc.no_direct_scanout` default to `true`?

### DISCUSSION THREAD

**Discussion by Chais** [hyprwm/Hyprland#2200] [General]
I've been having issues with video playback getting choppy on display 1 when display 2 actively refreshes (e.g. from moving the cursor over a different window (which updates borders), scrolling the Steam store (webview) or playing). I tracked the issue to wayland's enforced vsync and the option to enable direct scanout for fullscreen applications.
I was surprised that the flag to disable it (which would suggest that the default state is ON) defaults to `true`.
I would suggest to either,  
if there is a good reason to disable direct scanout by default, rename the flag to be positive (e.g. `direct_scanout` or `enable_direct_scanout`) and have it default to `false`.
Or preferably  
change the default for `no_direct_scanout` to `false`, as that seems to be the expected behaviour from a wayland compositor, at least in my limited understanding.

Cheers

**Comment by vaxerski** [comment]
there is a good reason to disable direct scanout by default.

**Reply by Chais** [reply]
Do I get to hear it?

**Reply by vaxerski** [reply]
nvidia: broken
amd: for many users, halving framerates when DS is active
intel: ditto

**Reply by Chais** [reply]
Thanks. Could we go with renaming the flag then, to avoid some of the confusion?

**Reply by vaxerski** [reply]
I'd rather not, as it's really annoying to end-users and I get 10 duplicate bug reports that "this opt doesnt work?///?????//?1/11/!/1!"

**Comment by Chais** [comment]
Fair enough.

**Comment by StichyHD** [comment]
So this is what i did is comment out  #no_direct_scanout = false do i recommend i can't say for sure but it fixed my bar tho and able to use the update.
